docs_user/modules/con_adoption-process-overview.adoc

```diff
@@ -24,7 +24,7 @@ endif::[]
 
 Post-adoption tasks::
 +
-* Optional: Run tempest to verify that the entire adoption process is working properly. For more information, see link:{defaultURL}/validating_and_troubleshooting_the_deployed_cloud/index[Validating and troubleshooting the deployed cloud].
-* Optional: Perform a minor update from RHEL 9.2 to 9.4. You can perform a minor update any time after you complete the adoption procedure. For more information, see link:{defaultURL}/updating_your_environment_to_the_latest_maintenance_release/index[Updating your environment to the latest maintenance release].
+* Optional: Run tempest to verify that the entire adoption process is working properly. For more information, see link:https://docs.redhat.com/en/documentation/red_hat_openstack_services_on_openshift/18.0/html/validating_and_troubleshooting_the_deployed_cloud/index[Validating and troubleshooting the deployed cloud].
+* After adoption, {rhos_long} data plane nodes run Red Hat Enterprise Linux (RHEL) 9.2. The data plane nodes can remain on RHEL 9.2; however, you must perform a system update for full functionality. You can perform a system update any time after you complete the adoption procedure. For more information about updating your environment, see link:https://docs.redhat.com/en/documentation/red_hat_openstack_services_on_openshift/18.0/html/updating_your_environment_to_the_latest_maintenance_release/index[Updating your environment to the latest maintenance release].
```

Comment:
'for full functionality'

Can we rather say something more like 'to use the full feature set from the release' or perhaps 'for full feature functionality' ?

Reply:
In addition to enabling the full feature set, updating the data plane nodes to RHEL 9.4 aligns the environment with the maximum support lifecycle of Red Hat OpenStack Services on OpenShift. This could be called out as another reason to plan the system update after adoption.